zeitschrift-fur-kunstgeschichte.csl

```diff
@@ -126,8 +134,8 @@
                           <text macro="volume"/>
                         </group>
                         <names variable="editor">
+                          <name and="text" delimiter-precedes-et-al="never" delimiter-precedes-last="never"/>
                           <label form="verb-short" suffix=" "/>
```

Comment:
this is almost certainly wrong: it's odd to have a verb label *after* the name in the first place. But if this is what you want, you'll need to make sure there's a space between name and label. Right now they'd be mashed into each other.